### PR TITLE
Origin/add flags skip data load

### PR DESCRIPTION
**Pull request by sandeepmanocha**
Added feature to refresh views without loading the data

New parameter name is -recreateviews True

Required Parameters along with recreateview:
Project Name
Dataset Name
DB Version
Script Version

example:
sandeepmanocha$ python3 optimusprime.py -dataset sandeep -recreateviews True -projectname optimusprime-migrations -importcomment "test run sandeep" -dbversion "190" -collectionversion "2.0.3"

### CODE REVIEW

File: db_assessment/optimusprime.py [hunk 1]
```diff
@@ -65,11 +65,36 @@ def runMain(args):
...
+        gcpProjectName = str(args.projectname)
+        bqDataset = str(args.dataset)
+
+        if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
+            
+            transformersParameters['dbversion'] = str(args.dbversion)
+            transformersParameters['optimuscollectionversion'] = str(args.collectionversion)
+            transformersParameters['recreateviews'] = True # setting this as parameter to that we can handle recreation in import_db_assesment.createOptimusPrimeViewsTransformers. If already exists, it returns error by default
+            rulesAlreadyExecuted = [] # keeping it, since it is evaluated in rules engine. although keeping it blank for now as for recreate views this should not impact
+            print(transformersParameters)
```
Comment (ersantosnet) [inline db_assessment/optimusprime.py:81]: Can we omit this print so we can keep our output cleaner? (our output today is already too long)

Comment (ersantosnet) [inline db_assessment/optimusprime.py:81]: Can we omit this print?

File: db_assessment/optimusprime.py [hunk 2]
```diff
@@ -260,11 +285,33 @@ def argumentsParser():
...
+    # SM: 04/28/2022:[Bug#90] Recreate views without loading the data:
+    parser.add_argument("-recreateviews", default=False, help="Recreate views without loading the data (True/False)")
+
     # Execute the parse_args() method. Variable args is a namespace type
     args = parser.parse_args()
 
+    # SM: 04/28/2022:[Bug#90] Check project name and big query dataset name
+    if str(args.recreateviews).upper() == "TRUE":
+        # In case there is not dataset parameter set or with valid content in the arguments. It is required during view recreates
+        print('Checking required parameters for recreating views')
```
Comment (ersantosnet) [inline db_assessment/optimusprime.py:297]: Can we omit this print?

**Comment by sandeepmanocha** [comment]
creating new one, after cleaning print statements

**Pull request closed without merging**